src/components/Reviews/Review.js

```diff
@@ -35,8 +33,9 @@ export default function Review({ review, styles }) {
               className={styles.starRating}
               role='img'
               aria-label={`Rated ${review.rating} out of 5`}
+              style={{"--rating":review.rating}}
             >
-              <span style={{ width: `${review.rating * 20}%` }}>
+              <span style={{display: "none"}}>
```

Comment:
hm, this is not exactly the same thing for accessibility, looking into rolling it back

Reply:
rolled back